cmd/flags/auth.go

```diff
@@ -64,11 +59,58 @@ const (
 	authOIDCClaimGroupEnv      = "AUTH_OIDC_CLAIM_GROUPS"
 )
 
+func init() {
```

Comment:
init methods must be at the beggining of the file by convention if I am not mistaken

Reply:
Ok